src/js/controllers/mapController.ts

```diff
@@ -774,6 +780,69 @@ export class MapController {
       }
     }
   };
+
+  setPolygon = (setDMSForm: Array<SpecificDMSSection>): void => {
+    const simpleFillSymbol = {
+      type: 'simple-fill', // autocasts as new SimpleFillSymbol()
+      color: [240, 171, 0, 0.0],
+      outline: {
+        // autocasts as new SimpleLineSymbol()
+        color: [0, 255, 254],
+        width: 2
+      }
+    };
+
+    this._mapview.graphics.removeAll();
```

Comment:
@SwampGuzzler does firing `this._mapview.graphics.removeAll()` unintentially remove existing graphics from another feature? Could we create a Layer (using a graphic) and then add/remove specific layers as needed? Just flagging, thanks!